src/bin/tikv-ctl.rs

```diff
@@ -819,6 +869,10 @@ fn main() {
         let from_key = matches.value_of("from").map(|k| unescape(k));
         let to_key = matches.value_of("to").map(|k| unescape(k));
         debug_executor.compact(db_type, cf, from_key, to_key);
+    } else if let Some(matches) = matches.subcommand_matches("log-behind") {
+        let region = matches.value_of("region").unwrap().parse().unwrap();
+        let leader_host = matches.value_of("leader-host").unwrap();
```

Comment:
Can it be obtained from pd?

Reply:
I think for now it can't because `get_region_by_id` doesn't return who is leader. If we want to support this, we need to update kvproto and pd.

Reply:
Actually the response of `get_region` contains the leader information. And if it's still missing, it can fallback to iterate all the peers to find out who is leader.